src/App.js

```diff
@@ -8,6 +8,7 @@ import Testimonial from "./Components/Testimonial";
 import FAQS from "./Components/FAQS";
 import QuestionsForm from "./Components/QuestionsForm";
 import Footer from "./Components/Footer";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
No need to add this bootstrap is already added in index.html. Just need react-bootstrap in your AboutUs.js